aerospike.go

```diff
@@ -19,14 +20,16 @@ type AerospikeContainer struct {
 // RunContainer creates an instance of the Aerospike container type.
 func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*AerospikeContainer, error) {
```

Comment:
I'd advocate for deprecating this `RunContainer` function and use the new APIs: `Run(ctx, image string, opts ...)`

Would you like to do it in this PR?